packages/components/src/components/tooltip/_tooltip.scss

```diff
@@ -398,6 +399,28 @@
       }
     }
 
+    // Tooltips without interactive elements, need to be sequentially and click focusable so screen reader users
+    // get the tooltip's content and we focus the content container instead of the tooltip root so that
+    // focus-wrap doesn't get triggered
+    .#{$prefix}--tooltip__content[tabindex='0'] {
+      // 1 rem === .#{$prefix}--tooltip padding
+      margin: calc(-1rem + 2px);
```

Comment:
should we use a spacing token in place of `1rem`?

Reply:
the 1rem was a copy and paste directly from the tooltip padding property. If I change this one, I feel like I should also change that one since they need to stay in sync. Agree?

Reply:
yeah I think that would be fine as long as we kept the comments about why we're making those calculations